package.json

```diff
@@ -33,6 +34,9 @@
       ],
       "testIds": [
         "./dist/esm/tests/test-ids-utils.d.ts"
+      ],
+      "next": [
+        "./dist/esm/next/next.d.ts"
```

Comment:
d.ts file for next entry (needed for eslint to not throw error but not helps with autocomplete)